### PR TITLE
Improve calendar a11y

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-12428
affects: student
release note: Accessibility improvements

Test plan:
* Log in to Student app. Account: narmstrong, User: s
* Go to Calendar tab and turn on Voice Over
* Dec 9, 2019 should say "2 events"
* Dec 13, 2019 should say "1 event"

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1015509&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Student/Canvas/Calendar/UI/View/CalendarDayCell.swift [hunk 1]
```diff
@@ -128,6 +126,10 @@ open class CalendarDayCell: UICollectionViewCell {
         dateLabel.accessibilityLabel = dateLabel.text
             .flatMap { Int($0) }
             .map(CalendarDayCell.dayOfTheMonthA11yFormatter)
+        if eventCount > 0 {
+            let format = NSLocalizedString("d_events", bundle: .core, comment: "")
+            dateLabel.accessibilityValue = String.localizedStringWithFormat(format, eventCount)
+        }
```
Comment (cobbal) [inline Student/Canvas/Calendar/UI/View/CalendarDayCell.swift]: Should this get updated in the case eventCount = 0?

Comment (armstrongnate) [reply Student/Canvas/Calendar/UI/View/CalendarDayCell.swift]: oh yes it should. good call good call

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



#### Release Note: 
Accessibility improvements

#### Affected Apps: Student

[MBL-12428](https://instructure.atlassian.net/browse/MBL-12428)

<details><summary>:x: XCTest failed: StudentUnitTests/CalendarMonthViewControllerTests/testDayCellA11yValues</summary>


    Assertion Failure: CalendarMonthViewControllerTests.swift:70: XCTAssertEqual failed: ("nil") is not equal to ("Optional("2 events")")
    Assertion Failure: CalendarMonthViewControllerTests.swift:72: XCTAssertEqual failed: ("nil") is not equal to ("Optional("1 event")")
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cobbal** [comment]
Doing some manual VO QA, and I noticed the following (some may have been present beforehand)

1. Some of the weekdays are pronounced using abbreviations, like "sun" and "mon". Ideally they'd be "sunday", ...
2. The greyed out saturday and sunday dates say, "fifth, button", while the weekdays just say "sixth".
3. It says, "January, twentieth twenty", though it does say "December, two thousand nineteen". Is this an apple problem or are we putting a weird label in?
4. Infinite scroll still sometimes makes it hard to get back to the top sometimes. I think this was mentioned in the middle of the ticket somewhere.

**Pull request merged**